main.js

```diff
@@ -1 +1,15 @@
-// all your js goes here
+$(document).ready(function(){
+	$('form#add').on('change', '#image-upload', function() {
+		var canvas    = document.getElementById('image-workspace');
+		var	context   = canvas.getContext('2d');
+		var img = new Image;
+    	img.src = URL.createObjectURL(event.target.files[0]);
```

Comment:
lines 6-12 need indenting - at least here on github. sometimes the indentation in your editor may not exactly match what you see online. If that is the case, then don't worry about it.